src/style.css

```diff
@@ -1,3 +1,78 @@
-body {
+body,
+html {
   margin: 0;
+  padding: 0;
+}
+
+.header {
+  --link-сolor: #000;
+  --link-сolor-active: #00acdc;
+  box-sizing: border-box;
+  display: flex;
+  justify-content: space-between;
+  margin: auto;
+  flex-shrink: 0;
+  font-size: 0;
+  width: 1024px;
+}
+
+.logo {
+  margin: 10px auto 10px 50px;
+  align-self: center;
+  width: 40px;
+  height: 40px;
+  flex-shrink: 0;
+}
+
+.nav {
+  margin-right: 40px;
+  flex-shrink: 0;
+}
+
+.nav-menu {
+  display: flex;
+  list-style: none;
+  padding: 0;
+  margin: 0;
+}
+
+.nav__list {
+  position: relative;
+  font-family: Roboto, sans-serif;
+  text-transform: uppercase;
+  font-size: 12px;
+  padding: 0 10px;
+  margin: 0 10px;
+}
+
+a.nav__list {
+  display: inline-block;
+  text-decoration: none;
+  color: var(--link-сolor);
+  padding: 23px 0;
+}
+
+a:hover {
+  color: var(--link-сolor-active);
+}
+
+a.is-active {
+  color: var(--link-сolor-active);
+}
+
+a.is-active::after {
+  position: absolute;
+  content: "";
+  background-color: var(--link-сolor-active);
+  height: 4px;
+  width: 100%;
+  border-radius: 8px;
+  left: 0;
+  bottom: 0;
+}
+
+@media (min-width: 1200px) {
+  .header {
+    width: 1200px;
+  }
```

Comment:
- not fixed 
<img width="775" alt="Screenshot 2023-10-13 at 13 40 36" src="https://github.com/mate-academy/layout_moyo-header/assets/64746943/e14fa28e-9463-4c24-b168-e5eff8c1942e">